src/core/blueprint/operators/js.rs

```diff
@@ -0,0 +1,60 @@
+use crate::core::blueprint::FieldDefinition;
+use crate::core::config;
+use crate::core::config::Field;
+use crate::core::lambda::IO;
+use crate::core::try_fold::TryFold;
+use crate::core::valid::Valid;
+use crate::{ConfigModule, Expression, ValidationError, Validator};
+
+pub struct CompileJs<'a> {
+    pub name: &'a str,
+    pub script: &'a Option<String>,
+}
+
+pub fn compile_js(inputs: CompileJs) -> Valid<Expression, String> {
+    #[cfg(not(feature = "js"))]
+    return js_disabled(inputs);
+
+    #[cfg(feature = "js")]
+    js_enabled(inputs)
+}
+
+#[cfg(not(feature = "js"))]
+fn js_disabled(_: CompileJs) -> Valid<Expression, String> {
+    Valid::fail("js is disabled or unsupported on this platform".to_string())
+}
+
+#[cfg(feature = "js")]
+fn js_enabled(inputs: CompileJs) -> Valid<Expression, String> {
+    let name = inputs.name;
+    let quickjs = rquickjs::Runtime::new().unwrap();
+    let ctx = rquickjs::Context::full(&quickjs).unwrap();
+
+    Valid::from_option(inputs.script.as_ref(), "script is required".to_string()).and_then(
```

Comment:
Just validate if the config has a worker script linked. No need of using quickjs for this.